README.md

```diff
@@ -144,7 +173,7 @@ To serve this app to end users, it is recommended to use a WSGI server such as [
 With Gunicorn, the app can be served using the following command:
 
 ```bash
-gunicorn main:server
+gunicorn --bind :$PORT app:server
```

Comment:
I think it'd be clearer to use `--bind 0.0.0.0:$PORT` since that appears to be what leaving out the host part does.